### PR TITLE
Add Polish translation of contribute index page

### PR DESCRIPTION
**Pull request by mfilocha**
This is Polish translation of "[Contribute](https://kubernetes.io/docs/contribute/)" main page.

### CODE REVIEW

**Comment by kpucynski** [comment]
I think that we should describe what SIG is, but it may be in different place.
/lgtm

File: content/pl/docs/contribute/_index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Więcej informacji na temat zawartości dokumentacji Kubernetesa i jej stylu,
+znajdziesz w
+ [Opisie stylu dokumentacji](/docs/contribute/style/).
+
+{{% capture body %}}
+
+## Rodzaje uczestnictwa w procesie tworzenia dokumentacji
+
+- _Członek_ (_member_) organizacji Kubernetes, który [podpisał CLA](/docs/contribute/start#sign-the-cla)
+  i poświęcił swój czas i wysiłek na rzecz projektu. Dokument
```
Comment (nvtkaszpir) [inline content/pl/docs/contribute/_index.md]:
```suggestion
  i poświęcił swój czas oraz wysiłek na rzecz projektu. Dokument
```

File: content/pl/docs/contribute/_index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  [Członkostwo w organizacji](https://github.com/kubernetes/community/blob/master/community-membership.md)
+  zawiera szczegóły z tym związane.
+- _Recenzent_ (_reviewer_) SIG Docs to członek organizacji Kubernetes, który zgłosił
+  swoją chęć recenzowania propozycji zmian w dokumentacji (PR) i został do dodany
```
Comment (nvtkaszpir) [inline content/pl/docs/contribute/_index.md]:
```suggestion
  swoją chęć weryfikacji propozycji zmian w dokumentacji (PR) i został dodany
```

File: content/pl/docs/contribute/_index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  treści w imieniu organizacji Kubernetes.
+  Osoby zatwierdzające mogą również reprezentować SIG Docs na szerszym forum
+  społeczności Kubernetes.
+  Niektóre wymagania związane z tą rolą, jak na przykład koordynacja kolejnego wydania,
+  wymagają poświęcenia znacznej ilości czasu.
+
+## Sposoby współpracy przy tworzeniu dokumentacji
+
+Poniższa lista podzielona jest na rzeczy, które może robić każdy, te, które może
+robić członek organizacji Kubernetes i na takie, które wymagają wyższych uprawnień
```
Comment (nvtkaszpir) [inline content/pl/docs/contribute/_index.md]:
```suggestion
robić członek organizacji Kubernetes oraz na takie, które wymagają wyższych uprawnień
```

File: content/pl/docs/contribute/_index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+i znajomości procesów SIG Docs. W miarę postępującej współpracy, będziesz mógł lepiej
+zrozumieć niektóre narzędzia różne decyzje, które zostały wcześniej podjęte
```
Comment (nvtkaszpir) [inline content/pl/docs/contribute/_index.md]:
```suggestion
zrozumieć niektóre narzędzia czy decyzje, które zostały wcześniej podjęte
```

File: content/pl/docs/contribute/_index.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+- [Osoba zatwierdzająca](/docs/contribute/advanced/)
+  - Publikowanie dostarczonych treści poprzez zatwierdzanie propozycji zmian i włączanie ich do repozytorium
+  - Udział w pracach zespołu przygotowującego nowe wydanie Kubernetesa jako przedstawiciel zespołu dokumentacji
+  - Proponowanie ulepszeń wytycznych dotyczących stylu
+  - Proponowanie ulepszeń testowania dokumentacji
+  - Proponowanie ulepszeń strony Kubernetes lub innych narzędzi
+
+## Inne metody współpracy
+
+- Aby włączyć się w prace społeczności Kubernetes poprzez fora internetowe typu Twitter lub Stack Overflow lub dowiedzieć się więcej na temat organizowanych lokalnie spotkań i wydarzeń związanym z projektem, zajrzyj na stronę [społeczności Kubernetes](/community/).
```
Comment (nvtkaszpir) [inline content/pl/docs/contribute/_index.md]:
```suggestion
- Aby włączyć się w prace społeczności Kubernetes poprzez fora internetowe, typu Twitter czy Stack Overflow, lub dowiedzieć się więcej na temat organizowanych lokalnie spotkań i wydarzeń związanym z projektem, zajrzyj na stronę [społeczności Kubernetes](/community/).

**Pull request merged**